### PR TITLE
2019 2 3 pipeline fixes with s0undt3chs changes

### PR DESCRIPTION
**Pull request by bryceml**
### What does this PR do?
re-opening #55445 with pedro's changes

### CODE REVIEW

**Review by s0undt3ch — APPROVED** [review]

File: .ci/kitchen-centos7-py2-cloud [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+runTestSuite(
+    concurrent_builds: 0,
+    distro_name: 'centos',
+    distro_version: '7',
+    env: env,
+    golden_images_branch: 'master',
+    jenkins_slave_label: 'kitchen-slave',
+    kitchen_platforms_file: '/var/jenkins/workspace/nox-cloud-platforms.yml',
+    nox_env_name: 'runtests-zeromq',
```
Comment (s0undt3ch) [inline .ci/kitchen-centos7-py2-cloud]: `runtests-cloud`

File: .ci/kitchen-centos7-py3-cloud [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+runTestSuite(
+    concurrent_builds: 0,
+    distro_name: 'centos',
+    distro_version: '7',
+    env: env,
+    golden_images_branch: 'master',
+    jenkins_slave_label: 'kitchen-slave',
+    kitchen_platforms_file: '/var/jenkins/workspace/nox-cloud-platforms.yml',
+    nox_env_name: 'runtests-zeromq',
```
Comment (s0undt3ch) [inline .ci/kitchen-centos7-py3-cloud]: `runtests-cloud`

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: .ci/kitchen-centos7-py2-cloud (hunk 1 above)
Comment (bryceml) [reply .ci/kitchen-centos7-py2-cloud]: I'll have to fix that on master too then, must have missed that earlier.

Comment (s0undt3ch) [reply .ci/kitchen-centos7-py2-cloud]: If we merge the macos on nox PR, it fixes it.

**Review by s0undt3ch — APPROVED** [review]

**Review by Akm0d — APPROVED** [review]

**Review by cmcmarrow — APPROVED** [review]

**Pull request merged**